lib/partial/transmuxer.js

```diff
@@ -317,9 +315,7 @@ var Transmuxer = function(options) {
   };
 
   this.setBaseMediaDecodeTime = function(baseMediaDecodeTime) {
-    if (!options.keepOriginalTimestamps) {
```

Comment:
so basically always track BMDT regardless of this option which will either be a value we care about or zero, in which case it won't do anything?

Reply:
Exactly